third_party/2and3/datetimerange/__init__.pyi

```diff
@@ -0,0 +1,38 @@
+from typing import Any, Optional, Union, Iterable
+import datetime
+from dateutil.relativedelta import relativedelta
+
+class DateTimeRange:
+    NOT_A_TIME_STR: str = ...
+    start_time_format: str = ...
+    end_time_format: str = ...
+    is_output_elapse: bool = ...
+    separator: str = ...
+    def __init__(self, start_datetime: Optional[str], end_datetime: Optional[str], start_time_format: str = ..., end_time_format: str = ...) -> None: ...
```

Comment:
`start_datetime` and `end_datetime` have default values, so `= ...` is missing. (`Optional` is unfortunately slightly misnamed and does not indicate optional arguments.)